setup.py

```diff
@@ -260,13 +259,12 @@ def parse_setuppy_commands():
                      'bdist_wininst', 'bdist_msi', 'bdist_mpkg', 'build_src',
                      'bdist_egg')
 
-    for command in good_commands:
-        if command in args:
-            return True
+    if args in good_commands:
+        return True
 
     # The following commands are supported, but we need to show more
     # useful messages to the user
-    if 'install' in args:
+    if 'install' == args:
```

Comment:
`args` is a list of strings; how exactly can a `'install' in args` check mix up the likes of `['install']` and `['install_lib']`?